modules/devices/steamdeck/fde.nix

```diff
@@ -0,0 +1,105 @@
+{ config, lib, pkgs, utils, ... }:
+
+let
+  cfg = config.jovian.devices.steamdeck;
+in
+{
+  options = {
+    jovian.devices.steamdeck = {
+      fde = {
+        deckbd = {
+          enable = lib.mkOption {
+            default = cfg.enable;
+            defaultText = lib.literalExpression "config.jovian.devices.steamdeck.enable";
+            type = lib.types.bool;
+            description = ''
+              Whether to enable deckbd for initrd password entry.
+
+              You may need hold the option key for a second before entering your
```

Comment:
That's a bit unclear, why is it *may*?

If it's not consistent, it might be necessary to figure out why, and get that fixed. "Random" behaviour isn't really desired here.

And the *option key*, that's the `•••` key, right? It might be appropriate to also include the label of the key when describing it.

Reply:
It is consistent. The use of _may_ was in case we changed something later. Should I change it to _must_?

By "option key" I actually meant the one with three horizontal lines in the upper right. I'm not sure what the official name is.

Reply:
Oh, right, then yes having the location and pictogram described will be necessary. I totally got the wrong key :sweat_smile: